### DISCUSSION TITLE
Proprosed script for generator's `package.json`

### DISCUSSION THREAD

**Discussion by jordanjones243** [AlaskaAirlines/WC-Generator#557] [Announcements]
When working on component theme support, I ran into an issue in `auro-input` where `scss` files were not being compiled into `css` files upon build. Even when copy/pasting the scripts from other components that did do this, the `css` files were still not being generated. The proposed solution is for the `build:sass:component` command and the code is as follows:

```
"build:sass:component": "for file in src/*.scss; do npx sass --no-source-map \"$file:${file%.scss}.css\"; done"
```

**Comment by jordanjones243** [comment]
Link to PR with current change:
- https://github.com/AlaskaAirlines/auro-input/pull/317/commits/229edcd5e5e0776fbb1091f62dbab407dc489842#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

**Reply by blackfalcon** [reply]
@jordanjones243 can you provide more context as to this update? Is this related to an update in Sass? 

You are introducing a `for loop`, is this a documented recommendation from somewhere? 

Also with the use of `npx` you are negating a local dependency, is this intentional?
